homework/c_sharp_basics_3/Program.cs

```diff
@@ -0,0 +1,87 @@
+﻿new Homework_3().Task_3();
+class Homework_3
+{
+    public void Task_1()
+    {
+        Console.WriteLine("Enter number");
+        string youNumber = Console.ReadLine();
+        int number = int.Parse(youNumber);
+        if (number % 2 == 0 && number % 5 == 0)
+        {
+            Console.WriteLine("tutti-frutti");
+        }
+        else if (number % 2 == 0)
+        {
+            Console.WriteLine("tutti");
+        }
+        else if (number % 5 == 0)
+        {
+            Console.WriteLine("frutti");
+        }
+        else
+        {
+            Console.WriteLine("Error");
+        }
+    }
+    public void Task_2()
+    {
+        Console.WriteLine("Enter number");
+        string firstNumber = Console.ReadLine();
+        int number = int.Parse(firstNumber);
+        Console.WriteLine("Enter a number greater than the first number");
+        string nextNumber = Console.ReadLine();
+        int greaterNumber = int.Parse(nextNumber);
+        for (; number <= greaterNumber; number++)
+        {
+            if (number % 2 == 0 && number % 5 == 0)
+            {
+                Console.WriteLine("tutti-frutti");
+            }
+            else if (number % 2 == 0)
+            {
+                Console.WriteLine("tutti");
+            }
+            else if (number % 5 == 0)
+            {
+                Console.WriteLine("frutti");
+            }
+            else
+            {
+                Console.WriteLine($"Number {number} can’t be divided on 2 or 5");
+            }
+
+        }
+        
+    }
+    public void Task_3()
+    {
+        Console.WriteLine("Enter number");
+        string firstNumber = Console.ReadLine();
+        int number = int.Parse(firstNumber);
+        Console.WriteLine("Enter next number");
+        string newNumber = Console.ReadLine();
+        int nextNumber = int.Parse(newNumber);
+        int min = Math.Min(number, nextNumber);
+        int max = Math.Max(number, nextNumber);
+        while (min < max)
+        {
+            min++;
```

Comment:
Does it really need to increase min value here?

Reply:
I need to increase the min value by 1 in order to do all the checks for numbers in the range.